src/algorithms/dqns/qr_dqn.jl

```diff
@@ -0,0 +1,158 @@
+export QRDQNLearner
+
+mutable struct QRDQNLearner{
+    Tq<:AbstractApproximator,
+    Tt<:AbstractApproximator,
+    Tf,
+    R<:AbstractRNG,
+} <: AbstractLearner
+    approximator::Tq
+    target_approximator::Tt
+    loss_func::Tf
+    min_replay_history::Int
+    update_freq::Int
+    update_step::Int
+    target_update_freq::Int
+    sampler::NStepBatchSampler
+    ensemble_num::Int
```

Comment:
Maybe you could change `ensemble_num` to `quant_num`.

Reply:
Will do that.